src/lib/atemSocketChild.ts

```diff
@@ -349,7 +370,7 @@ export class AtemSocketChild {
 			for (let i = fromIndex; i < this._inFlight.length; i++) {
 				const sentPacket = this._inFlight[i]
 				if (sentPacket.packetId === fromId || !this._isPacketCoveredByAck(fromId, sentPacket.packetId)) {
-					sentPacket.lastSent = Date.now()
+					sentPacket.lastSent = performance.now()
```

Comment:
I remember someone said that `performance.now()` is slow. Perhaps we should do `const now = performance.now()` outside of the loop, to avoid calling it inside these short loops. Same goes for `_checkForRetransmit`.
It might also want a guard `if (this._inFlight.length) return`, to avoid calling it unnecessarily